packages/application-extension/src/index.tsx

```diff
@@ -371,7 +371,10 @@ const mainCommands: JupyterFrontEndPlugin<void> = {
       });
 
       commands.addCommand(CommandIDs.toggleSidebarWidget, {
-        label: trans.__('Toggle Sidebar Element'),
+        label: args =>
+          args.side == 'right'
+            ? trans.__('Toggle Element %1 in %2 Sidebar', args.index, 'Right')
+            : trans.__('Toggle Element %1 in %2 Sidebar', args.index, 'Left'),
```

Comment:
We need to embed "Right' (and 'Left') in the text as otherwise these would not get translated (argument are not translated).

```suggestion
            ? trans.__('Toggle Element %1 in Right Sidebar', args.index)
            : trans.__('Toggle Element %1 in Left Sidebar', args.index),
```

Reply:
I have adapted the review changes.